### DISCUSSION TITLE
Refind boot loader

### DISCUSSION THREAD

**Discussion by gomma91** [prasanthrangan/hyprdots#681] [Q&A]
Can I install hyprdots with refind as boot loader?

**Comment by akashgagda** [comment]
No bro grub is suggested

**Reply by gomma91** [reply]
😢

**Comment by ghost** [comment]
Maybe, if after you install you want to swap with grub with something else, all power to you, it's your computer. The [ArchWiki rEFind page](https://wiki.archlinux.org/title/REFInd) looks like it gives the information you need, but other than that you won't get much help here.

**Reply by gomma91** [reply]
I already tried to switch grub with refind, but when i tried to log in hyprland crushed.

**Reply by ghost** [reply]
I don't know then, that might not even be an issue with us. I've never heard of rEFInd until now so i can't help you, sorry.

**Comment by kRHYME7** [comment]
I guess you can
